.circleci/config.yml

```diff
@@ -0,0 +1,125 @@
+version: 2.1
+workflows:
```

Comment:
What is node version we are using? Can we specify node version?

Reply:
By default it's v16 but I switched it to v14. We can specify build matrices to test many versions and change the version more easily if we use docker, but I'm using a machine executor because it's faster with the zk stuff.